tests/integration/checkpoint.bats

```diff
@@ -283,6 +284,7 @@ function simple_cr() {
 
   # checkpoint the running container
   runc --criu "$CRIU" checkpoint --work-path ./work-dir test_busybox
+  cat ./work-dir/dump.log | grep -B 5 Error || true
   [ "$status" -eq 0 ]
```

Comment:
this check doesn't make a sense

Reply:
Why? It totally does. We check that runc returns 0.

Reply:
The check should be before `cat .... true`

Reply:
No, it should be there is it.

First, the command `runc` is a shell function which uses bats `run` statement, which saves the exit code into `$status` (i.e. it does `status=$?` somewhere). We can check `$status` any time later, unless it's rewritten by another `run`, which is not the case here.

Second, in case `runc checkpoint` fails, we want to see the actual error. This is the reason why we do `cat | grep` before checking the status.